sympy/parsing/tests/test_latex.py

```diff
@@ -84,95 +85,87 @@ def test_import():
-    (r"0*1", _Mul(0, 1)),
-    (r"1 \times 2 ", _Mul(1, 2)),
-    (r"x = y", Eq(x, y)),
-    (r"x \neq y", Ne(x, y)),
-    (r"x < y", Lt(x, y)),
-    (r"x > y", Gt(x, y)),
-    (r"x \leq y", Le(x, y)),
-    (r"x \geq y", Ge(x, y)),
-    (r"x \le y", Le(x, y)),
-    (r"x \ge y", Ge(x, y)),
-    (r"\lfloor x \rfloor", floor(x)),
-    (r"\lceil x \rceil", ceiling(x)),
-    (r"\langle x |", Bra('x')),
-    (r"| x \rangle", Ket('x')),
-    (r"\sin \theta", sin(theta)),
-    (r"\sin(\theta)", sin(theta)),
-    (r"\sin^{-1} a", asin(a)),
-    (r"\sin a \cos b", _Mul(sin(a), cos(b))),
-    (r"\sin \cos \theta", sin(cos(theta))),
-    (r"\sin(\cos \theta)", sin(cos(theta))),
+    (r"a'b+ab'", _Add(_Mul(Symbol("a'"), b), _Mul(a, Symbol("b'"))))
+]
+
+FRACTION_EXPRESSION_PAIRS = [
     (r"\frac{a}{b}", a / b),
     (r"\dfrac{a}{b}", a / b),
     (r"\tfrac{a}{b}", a / b),
     (r"\frac12", _Pow(2, -1)),
     (r"\frac12y", _Mul(_Pow(2, -1), y)),
     (r"\frac1234", _Mul(_Pow(2, -1), 34)),
     (r"\frac2{3}", _Mul(2, _Pow(3, -1))),
-    (r"\frac{\sin{x}}2", _Mul(sin(x), _Pow(2, -1))),
     (r"\frac{a + b}{c}", _Mul(a + b, _Pow(c, -1))),
-    (r"\frac{7}{3}", _Mul(7, _Pow(3, -1))),
-    (r"(\csc x)(\sec y)", csc(x)*sec(y)),
-    (r"\lim_{x \to 3} a", Limit(a, x, 3, dir='+-')),
-    (r"\lim_{x \rightarrow 3} a", Limit(a, x, 3, dir='+-')),
-    (r"\lim_{x \Rightarrow 3} a", Limit(a, x, 3, dir='+-')),
-    (r"\lim_{x \longrightarrow 3} a", Limit(a, x, 3, dir='+-')),
-    (r"\lim_{x \Longrightarrow 3} a", Limit(a, x, 3, dir='+-')),
-    (r"\lim_{x \to 3^{+}} a", Limit(a, x, 3, dir='+')),
-    (r"\lim_{x \to 3^{-}} a", Limit(a, x, 3, dir='-')),
-    (r"\lim_{x \to 3^+} a", Limit(a, x, 3, dir='+')),
-    (r"\lim_{x \to 3^-} a", Limit(a, x, 3, dir='-')),
-    (r"\infty", oo),
-    (r"\lim_{x \to \infty} \frac{1}{x}", Limit(_Pow(x, -1), x, oo)),
-    (r"\frac{d}{dx} x", Derivative(x, x)),
-    (r"\frac{d}{dt} x", Derivative(x, t)),
-    (r"f(x)", f(x)),
-    (r"f(x, y)", f(x, y)),
-    (r"f(x, y, z)", f(x, y, z)),
-    (r"f'_1(x)", Function("f_{1}'")(x)),
-    (r"f_{1}''(x+y)", Function("f_{1}''")(x+y)),
-    (r"\frac{d f(x)}{dx}", Derivative(f(x), x)),
-    (r"\frac{d\theta(x)}{dx}", Derivative(Function('theta')(x), x)),
-    (r"x \neq y", Unequality(x, y)),
-    (r"|x|", _Abs(x)),
-    (r"||x||", _Abs(Abs(x))),
-    (r"|x||y|", _Abs(x)*_Abs(y)),
-    (r"||x||y||", _Abs(_Abs(x)*_Abs(y))),
-    (r"\pi^{|xy|}", Symbol('pi')**_Abs(x*y)),
+    (r"\frac{7}{3}", _Mul(7, _Pow(3, -1)))
+]
+
+RELATION_EXPRESSION_PAIRS = [
+    (r"x = y", Eq(x, y)),
+    (r"x \neq y", Ne(x, y)),
+    (r"x < y", Lt(x, y)),
+    (r"x > y", Gt(x, y)),
+    (r"x \leq y", Le(x, y)),
+    (r"x \geq y", Ge(x, y)),
+    (r"x \le y", Le(x, y)),
+    (r"x \ge y", Ge(x, y)),
+    (r"x < y", StrictLessThan(x, y)),
+    (r"x \leq y", LessThan(x, y)),
+    (r"x > y", StrictGreaterThan(x, y)),
+    (r"x \geq y", GreaterThan(x, y)),
+    (r"x \neq y", Unequality(x, y)), # same as 2nd one in the list
```

Comment:
This test is the same as the one on line 141, but the expected SymPy string is different. Which one should we keep?